addon/services/user-activity.js

```diff
@@ -38,6 +40,7 @@ export default Service.extend(Evented, {
       this.get('scrollActivity').on('scroll', this, this._handleScroll);
     } else if (this._eventsListened.indexOf(eventName) === -1) {
       this._eventsListened.push(eventName);
+      if (this.get('_isFastBoot')) { return; }
```

Comment:
We should probably move this check higher...I'm not sure it makes sense for us to mark that an event is being listened to (via `push`) if we're not actually going to call `addEventListener`